src/components/core/geoChart/legends/PushpinSizeLegend.tsx

```diff
@@ -11,14 +12,16 @@ export interface IPushpinSizeLegendProps {
 }
 
 export default function PushpinSizeLegend(props: IPushpinSizeLegendProps): JSX.Element {
-    const { sizes, format, numericSymbols = [], measureName } = props;
-    const minValue: number = Math.min(...sizes);
-    const maxValue: number = Math.max(...sizes);
+    const { sizes = [], format, numericSymbols = [], measureName } = props;
+    const sizeData = sizes.filter(isFinite);
+    const { min: minValue, max: maxValue } = getMinMax(sizeData);
+
```

Comment:
should we just :
min = Math.min (...sizeData),
....
because I see in getMinMax has duplicated to check isFinite.

Reply:
Fixed